src/main/java/org/cdc/gov/sdp/aphl/AIMSHeaderProcessor.java

```diff
@@ -1,4 +1,4 @@
-package org.cdc.gov.sdp;
+package org.cdc.gov.sdp.aphl;
```

Comment:
The package change here wasn't propagated to all of the test context files.  Please update them to either reflect the accurate path or to remove references to the AIMSHeaderProcessor.